composer.json

```diff
@@ -28,7 +27,8 @@
         "symfony/web-profiler-bundle": "^4.4",
         "twig/twig": "^2.0",
         "webmozart/assert": "^1.8",
-        "zircote/swagger-php": "^3.0"
+        "zircote/swagger-php": "^3.0",
+        "centreon/centreon-test-lib": "dev-MON-6465"
```

Comment:
when you will reset to `dev-master`, don't forget to reset alphabetical order too

Reply:
indeed